wp-content/plugins/core/src/Theme/Config/Image_Sizes.php

```diff
@@ -19,7 +19,10 @@ class Image_Sizes {
 	public const SOCIAL_SHARE_OPENGRAPH = 'social-share-opengraph';
 	public const SOCIAL_SHARE_TWITTER   = 'social-share-twitter';
 
-	private $sizes = [
+	/**
+	 * @var array<string, mixed>
```

Comment:
This comment format differs from previous where I've seen `[]string|mixed`

Reply:
I've only ever seen `string[]`, which would symbolize an array of strings, e.g. 

![image](https://user-images.githubusercontent.com/1066195/148469574-cd06e372-e8b7-48eb-8bf4-251c27bdf26e.png)

These are arrays with strings as keys, and mixed data as values (ints/bools).

I was being a bit lazy, so technically this should read `array<string, int|bool>` and now PhpStorm can warn you if you're using it for something other than what it's used for, because it now knows the exact type:

![image](https://user-images.githubusercontent.com/1066195/148469770-37b0e7ae-5d68-4ea4-b103-073f3f4de54f.png)